### PR TITLE
Fix packaging

### PR DESCRIPTION
**Pull request by grzegorz8**

### CODE REVIEW

File: tests/test_s3.py [hunk 1]
```diff
@@ -1,11 +1,9 @@
-import sys
 import unittest
 
 import boto3
 from moto import mock_s3
 
-sys.path.insert(0, "../")
-from s3 import get_latest_object  # noqa: E402
+from flink_sql_runner.s3 import get_latest_object  # noqa: E402
```
Comment (andrzejdackiewicz) [inline tests/test_s3.py:6]: Skoro już nie robimy sys.path.insert ... to mozna #noqa: E402 usunąć :)

File: tests/test_deploy_job.py [hunk 2]
```diff
@@ -9,13 +8,14 @@
 import boto3
 from moto import mock_s3
 
-from .test_s3_utils import put_object
+from flink_sql_runner.deploy_job import EmrJobRunner  # noqa: 402
```
Comment (andrzejdackiewicz) [inline tests/test_deploy_job.py]: We can remove # noqa: 402

File: tests/test_s3.py (hunk 1 above)
Comment (andrzejdackiewicz) [inline tests/test_s3.py]: Since we no longer perform sys.path.insert then we can remove #noqa: E402 :)

**Review by andrzejdackiewicz — CHANGES_REQUESTED** [review]

File: tests/test_s3.py (hunk 1 above)
Comment (grzegorz8) [reply tests/test_s3.py]: done

File: tests/test_deploy_job.py (hunk 2 above)
Comment (grzegorz8) [reply tests/test_deploy_job.py]: done

**Review by moss2k13 — APPROVED** [review]

**Review by andrzejdackiewicz — APPROVED** [review]
LGTM :)

**Pull request merged**